NetUtils.py

```diff
@@ -99,13 +99,24 @@ def _scan_for_TypedTuples(obj: typing.Any) -> typing.Any:
         data = obj._asdict()
         data["class"] = obj.__class__.__name__
         return data
-    if isinstance(obj, (tuple, list, set, frozenset)):
+    if isinstance(obj, (tuple, list, set)):
```

Comment:
Why do we not want `frozenset` here?

Reply:
`frozenset` is not a subclass of `set`. To match both `set` and `frozenset`, `typing.AbstractSet` is common to see in type hints, but it's slower to use in `isinstance` due to it being a `Protocol`, so I think reverting this change is best.

```python
>>> isinstance(frozenset(), (tuple, list, set))
False
>>> isinstance(frozenset(), (tuple, list, typing.AbstractSet))
True
```